content/en/user-guide/integrations/crossplane/index.md

```diff
@@ -0,0 +1,226 @@
+  * Alternatively, you can [create a local EKS cluster](https://docs.localstack.cloud/user-guide/aws/elastic-kubernetes-service/#create-an-embedded-kubernetes-cluster) in LocalStack directly, which will spin up a light-weight embedded `k3d` Kubernetes cluster in your Docker environment
+* The [`helm`](https://helm.sh) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) command-line clients installed
+
+Once your `kubectl` is configured to point to the local Kubernetes cluster, we first install Crossplane via `helm`:
+{{<command>}}
+$ helm repo add crossplane-stable https://charts.crossplane.io/stable
+$ helm repo update
+$ helm install crossplane crossplane-stable/crossplane --namespace crossplane-system --create-namespace
+{{</command>}}
+
+The installation may take a few minutes. In parallel, we can install the `crossplane` command-line extensions for `kubectl`:
+{{<command>}}
+$ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | bash
+...
+$ sudo mv kubectl-crossplane /usr/local/bin
+{{</command>}}
+
+To confirm that the installation was successful, we can run these `kubectl` commands, which should yield output similar to the following:
+{{<command>}}
+$ kubectl crossplane --version
+v1.13.2
+
+$ kubectl get crds | grep crossplane
+compositions.apiextensions.crossplane.io                     2023-09-03T11:30:36Z
+configurations.pkg.crossplane.io                             2023-09-03T11:30:36Z
+...
+{{</command>}}
+
+### Step 2: Installing the Crossplane AWS Provider
+
+Once the basic Crossplane installation is running properly, we can proceed with installing the AWS provider.
+Newer versions of Crossplane promote the use of [provider families](https://docs.upbound.io/providers/provider-families), which are collections of providers for different groups of resources.
+For example, there is a separate provider for each individual AWS service (like S3, SQS, Lambda, etc), and in addition provider family provides shared resources for common configuration of all services (e.g., credentials, etc).
+
+In the following, we first install the AWS provider for S3.
+Please note that you can copy/paste the entire multi-line command below into your terminal:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.40.0
+EOF
+{{</command>}}
+
+We also install the AWS provider for SQS:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws-sqs
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-sqs:v0.40.0
+EOF
+{{</command>}}
+
+After some time, the providers should get into healthy state, which can be confirmed via `kubectl get providers`:
+{{<command>}}
+$ kubectl get providers
+NAME                          INSTALLED   HEALTHY   PACKAGE                                               AGE
```

Comment:
Move output to non-command section so copy-paste works better?

Reply:
Will be handled by the changes from @webdev51 mentioned above.. 👍 (same for other comments in this PR)